src/bindgen/ir/function.rs

```diff
@@ -367,16 +367,16 @@ impl SynFnArgHelpers for syn::FnArg {
                         ))
                     }
                 };
-                let ty = match Type::load(ty)? {
+                let ir_ty = match Type::load(ty)? {
```

Comment:
changes in this file are unnecessary, restore it

Reply:
Fixed